bin/passenger-recycler

```diff
@@ -41,31 +41,39 @@ def process_status?(pid)
 end
 
 require 'phusion_passenger'
+PhusionPassenger.locate_directories
 require 'phusion_passenger/platform_info'
 require 'phusion_passenger/platform_info/ruby'
 require 'phusion_passenger/admin_tools/memory_stats'
-PhusionPassenger.locate_directories
```

Comment:
This change needs to be tested on SCL, I remember this was pain to get it working on both SCL and non-SCL.

Reply:
Well, I know I need it for the script to work at all. I get strange issues without this, like you can't load Passenger libraries because it can't find it :)

'bin/passenger-recycler'

    require 'phusion_passenger/platform_info'

'phusion_passenger/platform_info.rb'

    PhusionPassenger.require_passenger_lib 'utils/tmpio'

= fail, without it...

Reply:
No problem, this was a message for myself to test this with and without SCL.